src/services/auth/updatepassword.service.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import { AxiosError } from "axios";
+import { ENVIRONMENT } from "../../config/environment";
+
+type UpdatepasswordRequest = {
+  oldPassword: string;
+  newPassword: string;
+  token: string;
+};
+
+type UpdatepasswordResponse = {
+  msg: string;
+  success: boolean;
+};
+
+export const updatepasswordService = async (
+  req: UpdatepasswordRequest
+): Promise<UpdatepasswordResponse> => {
+  try {
+    const response = await axios.patch(
+      `${ENVIRONMENT.PROXY_BASE_URL}/account/password`,
+      req,
+      {
+        headers: {
+          Authorization: `Bearer ${req.token}`
+        }
+      }
+    );
+
+    if (response.status !== 200) {
```

Comment:
I think this `if-else` statement is not needed. If the response does not have a `200` status code an error will be thrown and caught by the `catch`.

Reply:
Done